docs/devGuide/projectManagement.md

```diff
@@ -71,81 +71,47 @@ For general best practices, refer to the guide [_Working with PRs_ @SE-EDU](http
 * You have the rights to push to master branch on [MarkBind's repository](https://github.com/MarkBind/markbind), and also to make new releases.
 * You have the rights on [npm](https://www.npmjs.com/) to make a new release.<br>
   You need to login to npm on your terminal with `npm adduser` first, before you can publish packages to npm.
+* Navigate to the root markbind directory before you start.
 </box>
 
 ### Building the UI components library
 
-1. **Navigate to the components** directory by running `cd frontend/components` in the root directory of MarkBind.
-1. **Update dependencies** by running `npm install`
-1. **Build the assets** by executing `npm run build`, after which you should see changes in `frontend/components/dist/vue-strap.min.js`.
+1. **Start with a "clean slate"** by running `npm run ci:all` or `npm run ciwin:all`, as this may affect the generated bundle.
+1. **Build the bundle** by executing `npm run build:components` in the root directory, after which you should see changes in `frontend/components/dist/components.min.js`.
 
    <box type="info" seamless>
 
-   Building vue-strap is optional if there's no changes to the components since the last release of MarkBind. If there are no changes to the three files mentioned above, skip to the "Building MarkBind" section.
+   Building the UI components bundle is optional if there's no changes to the `components.min.js` since the last release of MarkBind.
    </box>
    <box type="info" seamless>
 
-   Do take a peek at the diff for `vue-strap.min.js` to see if there's any strange changes (e.g. the version of jQuery changes even though no one upgraded it).
+   Do take a peek at the diff for `components.min.js` to see if there's any strange changes (e.g. the version of jQuery changes even though no one upgraded it).
    </box>
 
-1. **Commit** the new changes (`git commit -am 'Update dist'`)
```

Comment:
not too familiar with the release workflow;
since it's all in one repository now, should we lump it into one commit though?